sklearn/metrics/tests/test_common.py

```diff
@@ -1246,3 +1250,26 @@ def test_thresholded_multilabel_multioutput_permutations_invariance(name):
 
         current_score = metric(y_true_perm, y_score_perm)
         assert_almost_equal(score, current_score)
+
+
+@pytest.mark.parametrize(
+    'name',
+    set(THRESHOLDED_METRICS) - METRIC_UNDEFINED_BINARY_MULTICLASS)
+def test_multiclass_score_permutation_invariance(name):
```

Comment:
should it be clearer in the name that this applies only to thresholded (score-based) metrics?